go.mod

```diff
@@ -11,13 +11,11 @@ require (
 	github.com/getkin/kin-openapi v0.14.0
 	github.com/go-chi/chi v4.0.2+incompatible
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
-	github.com/go-sql-driver/mysql v1.4.1 // indirect
+	github.com/go-sql-driver/mysql v1.5.0 // indirect
```

Comment:
I know it has nothing to do with us but what in a world would require the mysql driver in our dependencies?